index.html

```diff
@@ -385,23 +384,21 @@ <h4>Primary Language</h4>
 
           <section>
             <h4>Script Events</h4>
-            <p>The <dfn>Script Events</dfn> part of the model provides the list of intervals corresponding to dialogue, on screen text, or descriptions.</p>
+            <p>A <a>DAPT Script</a> MAY contain zero or more <a>Script Event</a> objects, each corresponding to dialogue, on screen text, or descriptions for a given time interval.</p>
           </section>
 
           <section>
             <h4>Characters</h4>
-            <p>The <dfn>Characters</dfn> part of the model provides the list of characters referenced in the <a>Script Events</a>.</p>
+            <p>A <a>DAPT Script</a> MAY contain zero or more <a>Character</a> objects, each describing a character referenced in a <a>Script Event</a>.</p>
```

Comment:
We cannot require that each Character object is referenced in a Script Event, since a workflow might involve importing characters from some other source before any transcription has happened.

```suggestion
            <p>A <a>DAPT Script</a> MAY contain zero or more <a>Character</a> objects, each describing a character that can be referenced by a <a>Script Event</a>.</p>
```

Reply:
In a finished document, I think it would be simpler if only those characters that are used are present in a document. But maybe that's imposing too much. Should be a 'should'.

Reply:
Ok to use your suggestion and revisit when discussing https://github.com/w3c/dapt/issues/84